src/components/lib/cos_defcompinfo_api.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2016, Phani Gadepalli and Gabriel Parmer, GWU, gparmer@gwu.edu.
+ *
+ * This uses a two clause BSD License.
+ */
+
+#include <cos_defcompinfo_api.h>
+
+static struct cos_defcompinfo curr_defci;
+
+struct cos_defcompinfo *
+cos_defcompinfo_curr_get(void)
+{ return &curr_defci; }
```

Comment:
I'm sorry I was wrong. I can neither assert here nor add that comment, because we extract/get the `cos_compinfo` struct from this to call `cos_meminfo_init()`, which is not part of the `cos_defcominfo_init()`.